libs/vm_monitor/src/dispatcher.rs

```diff
@@ -118,7 +118,15 @@ impl Dispatcher {
     /// serialize the wrong thing and send it, since `self.sink.send` will take
     /// any string.
     pub async fn send(&mut self, message: OutboundMsg) -> anyhow::Result<()> {
-        info!(?message, "sending message");
+        match &message.inner {
+            crate::protocol::OutboundMsgKind::HealthCheck { .. } => {
+                debug!(?message, "sending message");
+            }
+            _ => {
+                info!(?message, "sending message");
+            }
+        }
```

Comment:
thoughts on simplifying (here and elsewhere) with `if let ...` and/or `if matches!(...)` ?

just thinking ways to make this less verbose.